test/unit/change_trust_asset_test.js

```diff
@@ -0,0 +1,355 @@
+      expect(() => StellarBase.ChangeTrustAsset.native()).to.not.throw;
+    });
+  });
+
+  describe('getCode()', function() {
+    it('returns a code for a native asset', function() {
+      const asset = StellarBase.ChangeTrustAsset.native();
+      expect(asset.getCode()).to.be.equal('XLM');
+    });
+
+    it('returns a code for a non-native asset', function() {
+      const asset = StellarBase.ChangeTrustAsset.creditAsset(
+        'USD',
+        'GCEZWKCA5VLDNRLN3RPRJMRZOX3Z6G5CHCGSNFHEYVXM3XOJMDS674JZ'
+      );
+      expect(asset.getCode()).to.be.equal('USD');
+    });
+
+    it('returns undefined code for a liquidity pool asset', function() {
+      const asset = StellarBase.ChangeTrustAsset.liquidityPoolAsset({
+        assetA,
+        assetB,
+        fee
+      });
+      expect(asset.getCode()).to.be.undefined;
+    });
+  });
+
+  describe('getIssuer()', function() {
+    it('returns undefined issuer for a native asset', function() {
+      const asset = StellarBase.ChangeTrustAsset.native();
+      expect(asset.getIssuer()).to.be.undefined;
+    });
+
+    it('returns an issuer for a non-native asset', function() {
+      const asset = StellarBase.ChangeTrustAsset.creditAsset(
+        'USD',
+        'GCEZWKCA5VLDNRLN3RPRJMRZOX3Z6G5CHCGSNFHEYVXM3XOJMDS674JZ'
+      );
+      expect(asset.getIssuer()).to.be.equal(
+        'GCEZWKCA5VLDNRLN3RPRJMRZOX3Z6G5CHCGSNFHEYVXM3XOJMDS674JZ'
+      );
+    });
+
+    it('returns undefined issuer for a liquidity pool asset', function() {
+      const asset = StellarBase.ChangeTrustAsset.liquidityPoolAsset({
+        assetA,
+        assetB,
+        fee
+      });
+      expect(asset.getIssuer()).to.be.undefined;
+    });
+  });
+
+  describe('getLiquidityPoolParameters()', function() {
+    it('returns undefined liquidity pool params for a native asset', function() {
+      const asset = StellarBase.ChangeTrustAsset.native();
+      expect(asset.getLiquidityPoolParameters()).to.be.undefined;
+    });
+
+    it('returns undefined liquidity pool params for a non-native asset', function() {
+      const asset = StellarBase.ChangeTrustAsset.creditAsset(
+        'USD',
+        'GCEZWKCA5VLDNRLN3RPRJMRZOX3Z6G5CHCGSNFHEYVXM3XOJMDS674JZ'
+      );
+      expect(asset.getLiquidityPoolParameters()).to.be.undefined;
+    });
+
+    it('returns liquidity pool parameters for a liquidity pool asset', function() {
+      const asset = StellarBase.ChangeTrustAsset.liquidityPoolAsset({
+        assetA,
+        assetB,
+        fee
+      });
+      const gotPoolParams = asset.getLiquidityPoolParameters();
+      expect(gotPoolParams.assetA).to.be.equal(assetA);
+      expect(gotPoolParams.assetB).to.be.equal(assetB);
+      expect(gotPoolParams.fee).to.be.equal(fee);
+    });
+  });
+
+  describe('getAssetType()', function() {
+    it('returns "native" for native assets', function() {
+      const asset = StellarBase.ChangeTrustAsset.native();
+      expect(asset.getAssetType()).to.eq('native');
```

Comment:
nit: why `.to.be.equal(...)` above and `.to.eq(...)` here?

Reply:
No particular reason. Seems like this test suite allows many different ways of writing and I got multiple ones due to cp[y & paste. I'll update the code to use only one variation.

Reply:
Fixed in https://github.com/stellar/js-stellar-base/pull/454/commits/fbec065664b70070037e631190ddb1e9157391a8.